postgresql/db-rel-status.yml

```diff
@@ -0,0 +1,85 @@
+type: db
+data:
+  query: SELECT * FROM pg_stat_user_tables;
+  dbUrl: jdbc:postgresql://${POSTGRESQL_DB_HOST_PORT}
+  dbDriverClass: org.postgresql.Driver
+  dbUser: ${POSTGRESQL_DB_USER}
+  dbPassword: ${POSTGRESQL_DB_PASSWORD}
+  dbAdditionalConnectionParams: ${POSTGRESQL_DB_ADDITIONAL_PARAMS}
+
+observation:
+  - name: usertables
+    metricNamespace: postgresql
+
+    metric:
+      - name: tables.user.table
+        source: relname
+        type: text
+        send: false
+        label: Table name
+
+      - name: tables.user.schema
+        source: schemaname
+        type: text
+        send: false
+        label: Schema name
+
+      - name: tables.user.sequential.scan
```

Comment:
Typically would be other way around - `tables.scan.sequential`. And other metrics are missing `scan` part.